sigpro/transformations/frequency/fftfreq.py

```diff
@@ -16,6 +16,7 @@ def fft_freq(amplitude_values, sampling_frequency):
             * `amplitude_values (numpy.ndarray)`
             * `frequency_values (numpy.ndarray)`
     """
-    frequency_values = np.fft.fftfreq(len(amplitude_values), 1 / sampling_frequency)
+    # frequency_values = np.fft.fftfreq(len(amplitude_values), 1 / sampling_frequency)
+    frequency_values = np.arange(0, len(amplitude_values)) * sampling_frequency
```

Comment:
Shouldn't it be divided by `len(amplitude_values)`?
![Screenshot 2024-11-14 at 9 53 28 AM](https://github.com/user-attachments/assets/0ee78a3d-a366-4dbd-a9c7-5b4d51fb123e)

Reply:
Theoretically the end time is:
End Time = Timestamp + (Length of Values * sampling frequency )
So the frequencies values are 
np.arange(0, N)* sampling frequency